### DISCUSSION TITLE
note/vuepress-theme-plume/quick-start/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/pengzhanbo.cn#31] [Blog-Comment]
# note/vuepress-theme-plume/quick-start/

依赖环境
Node.js v12+ (https://nodejs.org/en/); 
Yarn v1 classic (https://classic.yarnpkg.com/en/); ::: tip 提示
使用 pnpm (https://pnpm.io/zh/) 时，你需要在 .npmrc 文件中设置 shamefully-hoist=tru...

https://pengzhanbo.cn/note/vuepress-theme-plume/quick-start/

**Comment by Jetive** [comment]
最后一行所说的 **首页配置** 和 **导航栏配置** 在哪里？

**Reply by pengzhanbo** [reply]
[首页配置](https://pengzhanbo.cn/note/vuepress-theme-plume/page-config/#home)
[导航栏配置](https://pengzhanbo.cn/note/vuepress-theme-plume/theme-config/#navbar)
